tests/workflows/test_dependency_binaries_manifest.py

```
@@ -0,0 +1,431 @@
+    """
+    script_path = (
+        Path(__file__).resolve().parents[2]
+        / "installer"
+        / "scripts"
+        / "dependency_binaries_manifest.py"
+    )
+    spec = importlib.util.spec_from_file_location(
+        "dependency_binaries_manifest", script_path
+    )
+    if spec is None or spec.loader is None:
+        raise ImportError(f"Failed to load module spec from {script_path}")
```

Ruff (0.15.9)

[warning] Avoid specifying long messages outside the exception class

(TRY003)